dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/DotRestHighLevelClientProvider.java

```diff
@@ -106,14 +110,24 @@ private BasicCredentialsProvider getCredentialsProvider() throws IOException, Ge
 
     private RestClientBuilder getClientBuilder(final BasicCredentialsProvider credentialsProvider) {
         final String esAuthType = getESAuthType();
-
+        final String esProtocol = Config.getStringProperty("ES_PROTOCOL", "https");
         final RestClientBuilder clientBuilder = RestClient
                 .builder(new HttpHost(Config.getStringProperty("ES_HOSTNAME", "127.0.0.1"),
-                        Config.getIntProperty("ES_PORT", 9200), sslContextFromPem != null ? "https" :"http"))
+                        Config.getIntProperty("ES_PORT", 9200), esProtocol))
                 .setHttpClientConfigCallback((httpClientBuilder) -> {
                     if (sslContextFromPem != null) {
                         httpClientBuilder
                                 .setSSLContext(sslContextFromPem);
+                    } else if ("https".equals(esProtocol)){
+                        try {
+                            httpClientBuilder.setSSLContext(SSLContexts
+                                    .custom().loadTrustMaterial(new TrustSelfSignedStrategy()).build());
+                        } catch (NoSuchAlgorithmException | KeyManagementException | KeyStoreException e) {
+                            Logger.error(this.getClass(),
+                                    "Error setting TrustSelfSignedStrategy for Elastic RestHighLevelClient",
+                                    e);
+                            throw new DotRuntimeException(e);
+                        }
```

Comment:
is it possible do any test?

Reply:
how do you suggest to test it? The logic avoid sending certificates for validation when the ES connection is established